src/parser/jobs/mch/modules/Tincture.tsx

```diff
@@ -0,0 +1,66 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS, {Action} from 'data/ACTIONS'
+import Downtime from 'parser/core/modules/Downtime'
+import {dependency} from 'parser/core/Module'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import {Tincture} from 'parser/core/modules/Tincture'
+import React from 'react'
+import {BuffWindowState, BuffWindowTrackedAction} from 'parser/core/modules/BuffWindow'
+
+export default class MchTincture extends Tincture {
+	buffAction = ACTIONS.INFUSION_DEX
+
+	@dependency private downtime!: Downtime
+
+	trackedActions = {
+		icon: ACTIONS.INFUSION_DEX.icon,
+		actions: [
+			{
+				action: ACTIONS.WILDFIRE,
+				expectedPerWindow: 1,
+			},
+			{
+				action: ACTIONS.REASSEMBLE,
+				expectedPerWindow: 1,
+			},
+			{
+				action: ACTIONS.PILE_BUNKER,
+				expectedPerWindow: 1,
+			},
+			{
+				action: ACTIONS.DRILL,
+				expectedPerWindow: 2,
+			},
+		],
+		suggestionContent: <Trans id="mch.tincture.suggestions.trackedActions.content">
+			Try to cover as much damage as possible with your Infusions of Dexterity.
```

Comment:
Wording nit, this should be "Tinctures" not "Infusions". Infusions are soooooo last expansion.

Reply:
```suggestion
			Try to cover as much damage as possible with your Tinctures of Dexterity.
```